apps/src/templates/codeDocs/MethodSummaryTable.jsx

```diff
@@ -1,33 +1,29 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import EnhancedSafeMarkdown from '@cdo/apps/templates/EnhancedSafeMarkdown';
-import FontAwesome from '@cdo/apps/templates/FontAwesome';
 import color from '@cdo/apps/util/color';
 import {tableLayoutStyles} from '@cdo/apps/templates/tables/tableConstants';
-import {TextLink} from '@dsco_/link';
 
-export default function MethodSummaryTable({
-  methods,
-  programmingClassLink,
-  openLinkInNewTab = false
-}) {
+export default function MethodSummaryTable({methods, includeLink}) {
   return (
     <table style={{...tableLayoutStyles.table, width: '100%'}}>
       <tbody>
         {methods.map(method => (
           <tr key={method.key}>
             <td style={styles.method}>
               <h3>
-                {method.name}
-                {programmingClassLink && (
-                  <>
-                    {' '}
-                    <TextLink
-                      href={`${programmingClassLink}#method-${method.key}`}
-                      openInNewTab={openLinkInNewTab}
-                      icon={<FontAwesome icon="external-link" />}
-                    />
-                  </>
+                {includeLink ? (
```

Comment:
When wouldn't we want to include the link?

Reply:
I put this in in case we wanted a situation where we only showed the summary but not the details, but that may be overkill.

Reply:
I'll remove the includeLink option since for now we always want to show a link